README.md

```diff
@@ -228,6 +228,91 @@ func TestMain(m *testing.M) {
 }
 ```
 
+### Usage of a custom formatter
+
+You can implement a custom format when running godog. To achieve that, you need to implement a struct which implement the interface Formatter.
+
+See the following example :
+
+```go
+package fmtminimale
+
+func init() {
+	godog.Format("minimale", "Prints minimal status", minimaleFunc)
+}
+
+func minimaleFunc(suite string, out io.Writer) godog.Formatter {
+	return &minimale{
+		out:     out,
+		indent:  2,
+		started: time.Now(),
+    delayedMessageWriter: godog.NewDelayedMessagesWriter(),
+	}
+}
+
+type minimale struct {
+  delayedMessageWriter *godog.DelayedMessagesWriter
+	out     io.Writer
+	indent  int
+	started time.Time
+}
+
+func (m *minimale)Output() io.writer {
+  return m.delayedMessageWriter
+}
+
+func (m *minimale) Feature(f *gherkin.Feature, s, string, b, []byte) {
```

Comment:
there are many mistakes in this code sample. It is too simple to read the godoc and implement the formatter if they need it. so far everyone was able to manage it without questions.